fbgemm_gpu/codegen/training/backward/embedding_backward_split_template.cu

```diff
@@ -1085,6 +1175,7 @@ Tensor {{ embedding_cuda_op }}(
                     // Compute shared memory size for warp_per_row
                     int32_t num_warp_per_row_groups = kBackwardMaxThreads / kThreadGroupSize;
                     int32_t warp_per_row_smem_bytes = 0;
+                    auto blockSize = dim3(kThreadGroupSize, num_warp_per_row_groups);
```

Comment:
@q10 Consider moving this line after **if(kUseVecBlocking)** statement below. This should fix **test_backward_adagrad_large_dims** unit test